static/css/sass/dcr.scss

```diff
@@ -0,0 +1,261 @@
+@import url('https://fonts.googleapis.com/css?family=Libre+Baskerville:400i|Open+Sans:400,600');
+
+$unc-blue: #4B9CD3;
+$link-blue: #007FAE;
+$unc-navy: #13294B;
+$light-gray: #E1E1E1;
+$dark-gray: #767676;
+$black: #151515;
+
+@mixin mobile-margin {
+  @media only screen and (max-width: 1088px) {
+    margin-left: 16px;
+    margin-right: 16px;
+  }
+}
+
+* {
+  font-family: 'Open Sans', sans-serif;
+}
+
+body {
+}
+
+.logo-row {
+  background-color: $unc-blue;
+}
+
+.menu-row, .banner-row, .search-row, footer {
+  background-color: $unc-navy;
+}
+
+.info-row {
+  background-color: #F5F5F5;
+}
+
+.logo, .menu {
+  padding: 12px 0; // 1rem = 16px
+}
+
+.logo {
+  @include mobile-margin;
+  a {
+    display: flex;
+    align-items: center;
+    max-width: 550px;
+  }
+  img {
+    display: inline-block;
+    max-width: 250px;
+    width: 100%;
+    margin: 8px 0;
+  }
+  h1 {
+    display: inline-block;
+    margin-left: 24px;
+    font-family: 'Libre Baskerville', serif;
+    font-size: 1.25rem;
+    font-style: italic;
+    color: white;
+    line-height: 1.5;
+  }
+}
+
+.navbar {
+  box-shadow: 2px 2px 2px 0 #071730;
+  a {
+    color: white;
+  }
+
+  .is-active {
+    background-color: $unc-navy;
+  }
+}
+
+.banner-row {
+  background-image: url(../front/banner.jpg);
```

Comment:
This file doesn't appear to exist

Reply:
Hmm, this seems to load okay for me. The path is relative to where it ends up; as part of `static/css/cdr_access.css`

Reply:
On my VM it is resolving to http://localhost:8181/static/front/banner.jpg
The files in css/images/ tend to be sprite maps, maybe it would be more appropriate to put the file with the other images, in `static/images`?

Reply:
Hmm, yeah it might have been there at one point

Reply:
image was on the fcrepo3 branch but not the fcrepo4 branch